modules/ROOT/pages/7.3-release-notes.adoc

```diff
@@ -141,6 +141,13 @@ As a result, non-breaking spaces between different inline elements are now norma
 
 {productname} {release-version} also includes the following bug fix<es>:
 
+=== `preventDefault` on mousedown on toolbar buttons was causing misplaced focus bugs.
+// #TINY-10638
+
+In previous versions of {productname}, the `preventDefault` on mousedown in the toolbar button component caused focus issues. As a consequence, when clicking on different toolbar buttons, the focus was not updated, leaving the initial button highlighted.
+
+{productname} {release-version} addresses this issue by removing the `preventDefault`, which allows the focus to change as expected with user actions, such as mouse clicks or tab navigation.
```

Comment:
```suggestion
{productname} {release-version} addresses this issue by removing the `preventDefault`, allowing the focus to change as expected when users click on the image controls (ie. zoom in and out).
```

> change as expected with user actions.

this statement is not exactly correct, from the ticket, the user expected the focus (box-shadow) to change when clicking on the image and clicking on other buttons, this fix only addresses the focus to change when user clicks on the image control buttons